quokka/core/tests/test_models.py

```diff
@@ -1,8 +1,13 @@
 # coding: utf-8
+import sys
+
 from . import BaseTestCase
 
 from ..models import Channel, Config, CustomValue
 
+if sys.version_info.major == 3:
+    unicode = lambda x: u'{}'.format(x)
```

Comment:
I think is the better way (but the lambda approach is a good idea too)

http://lucumr.pocoo.org/2014/1/5/unicode-in-2-and-3/
_On the other hand on Python 2 we have two text types: str which for all intents and purposes is limited to ASCII + some undefined data above the 7 bit range, **unicode which is equivalent to the Python 3 str type** and one byte type bytearray which it inherited from Python 3._